docs/pages/includes/database-access/aws-create-db-configuration-with-var-protocol.mdx

```diff
@@ -0,0 +1,27 @@
+{{ partialSuffix="" }}
+
+Install Teleport on the host where you will run the Teleport Database Service:
+
+(!docs/pages/includes/install-linux.mdx!)
+
+On the node that is running the Database Service, create a configuration file
```

Comment:
I think it would be a lot simpler to recommend one method of creating a configuration and guiding readers toward the "Dynamic Resources" docs subsection intro at the end of each page.

The way I see it, each how-to guide should include a small number of paths that will work for most users (or that most users can follow as an example for their own use case), then direct users to guides that let them pick a more customized approach.

Reply:
> The way I see it, each how-to guide should include a small number of paths that will work for most users 

Agreed on small number of paths. However we do get ppl asking how to update existing config files or how to configure static databases. They may not be the majority cases but still generate some support load. So it's important to document them.

Doing these tabs is the easiest way I can think of. If we are to drop the other tabs here, what would you recommend so ppl on those paths can easily figure out what to do?

Thanks!

Reply:
What would you think of leaving only the "Modifying existing Database Service" content? I like the idea of showing all of the relevant configuration options within the guide as a way of educating users about the configuration file. It's also faster than the other two approaches, since it doesn't require running a binary, and users can copy the config into a user data script as shown in the [Deploy Teleport Agents with Terraform](https://goteleport.com/docs/agents/deploy-agents-terraform/) guide.

The "Next steps" section links to the docs on dynamic registration, so users should eventually learn about this technique for registering databases.

Reply:
But the  `teleport db configure create`  is our  recommended way to configure db service. 
By covering only  "Static Configuration" approach we will not guide to use `teleport db configure create` command and also will also introduce inconsistency with Self-hosted DB guides  where we guide users to use `teleport db configure create` command. 

I agree that showing static configuration is very valuable but I think that we should do that in addition to our default/recommended `teleport db configure create` flow.

Reply:
Showing the `teleport db configure create` command works for me! We could then include a paragraph/bullet point in the "Next steps" section that briefly tells the user how to edit an existing config file, pointing to the config reference and the "Teleport Agents" [section intro](https://goteleport.com/docs/agents/introduction/).

My main goal here is to have a single path for configuring the Database Service that satisfies 80% of users to avoid a ton of complexity in this guide. The other 20% can read our configuration reference etc.

Reply:
If we only show a single path with `teleport db configure create`, then it will be for dynamic resources. I will make this change if we agree. Probably there is no single path that will cover 80% of users. 

> We could then include a paragraph/bullet point in the "Next steps" section that briefly tells the user how to edit an existing config file, pointing to the config reference and the "Teleport Agents" [section intro](https://goteleport.com/docs/agents/introduction/).

"Next Steps" already links to the YAML reference. Maybe we can reword it to something like `To understand or update the YAML configuration, refer to the configuration reference.` ?

Reply:
That sounds good to me!